From f38af8c7f3a5a572ad56f64ba937bfc5cf21dfd1 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Wed, 26 Oct 2022 08:45:14 +0200
Subject: [PATCH 3/3] Fix incompatible pointer conversion

vomsd.cc:1454:19: error: incompatible integer to pointer conversion assigning to 'char *' from 'char'
        (*role) = '\0';
                  ^~~~
5 warnings and 1 error generated.
---
 src/server/vomsd.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/server/vomsd.cc b/src/server/vomsd.cc
index 700c5f1..8735dc9 100644
--- a/src/server/vomsd.cc
+++ b/src/server/vomsd.cc
@@ -1451,7 +1451,7 @@ static bool determine_group_and_role(std::string command, char *comm, char **gro
     case 'B':
       *role = strchr(string, ':');
       if (*role) {
-        (*role) = '\0';
+        (*role) = "\0";
         (*role)++;
       }
       break;
-- 
2.37.0 (Apple Git-136)

